ruby.md

```diff
@@ -4,29 +4,36 @@ Try your best to answer each question on your own before looking up the answer o
 
 
 #### 1. What is a method in Ruby? How is it different or similar to functions in JavaScript?
-
+a methond is ruby is a set of expressions that return values which is what a function does in JavaScript. 
 
 #### 2. What does it mean that a class inherits from another class? Try to explain Ruby inheritance. 
 
 
 [Your Answer]
+a set of values a class gets from another 
 
 
 [Googled Answer]
-
+a relationship between two classes. 
 
 #### 3. What is rspec and what is the general process for writing tests in RSpec?
 
 //Your Answer
+testing in ruby 
 
 //Googled Answer
-
+RSpec is a 'Domain Specific Language' (DSL) testing tool written in Ruby to test Ruby code. It is a behavior-driven development (BDD) framework which is extensively used in production applications.
+rspec car_spec.rb
 
 #### 4. Name three possible non-inheritance relationships between ruby objects? 
 
```

Comment:
Examples of non-inheritance relationships are has-many, belongs-to, has-one, etc